### PR TITLE
Clicking picture added into camera example 

### PR DESCRIPTION
**Pull request by yaki29**
Added a "click_pic()" function to take the picture and it even saves it in the format as almost all camera applications do in the format of "IMG_26-12-2016_11:30:23". This name tells that it is image file clicked on 26-12-2016 at 11:30:23.

### CODE REVIEW

File: examples/camera/main.py [hunk 1]
```diff
@@ -16,25 +16,44 @@
...
+class CameraClick(BoxLayout):
+    def click_pic(self):
+        '''
+        Function to capture the images and give them the names 
+        according to their captured time and date.
+        '''
+        camera = self.ids['camera']
+        timestr = time.strftime("%d-%m-%Y_%H:%M:%S")
+        camera.export_to_png("IMG_" + timestr)
+        print "Captured"
```
Comment (dessant) [inline examples/camera/main.py]: Crashes on py3.

File: examples/camera/main.py [hunk 2]
```diff
@@ -16,25 +16,44 @@
...
+        Button:
+            text: 'Capture'
+            size_hint_y: None
+            height: '48dp'
+            on_press: root.click_pic()
 '''
-
+)
+class CameraClick(BoxLayout):
+    def click_pic(self):
```
Comment (dessant) [inline examples/camera/main.py]: Please rename `click_pic` to `capture` or something more descriptive.

File: examples/camera/main.py [hunk 3]
```diff
@@ -16,25 +16,44 @@
...
-    ToggleButton:
-        text: 'Play'
-        on_press: camera.play = not camera.play
-        size_hint_y: None
-        height: '48dp'
+from kivy.uix.boxlayout import BoxLayout
+import time
+Builder.load_string('''
+<CameraClick>:
+    BoxLayout:
```
Comment (dessant) [inline examples/camera/main.py]: `CameraClick`  is already a BoxLayout, the additional nesting is not needed.

**Comment by rjt** [comment]
Colin : characters do not work as file names on Windows.  

YYYYmmdd-HHMMSS format provides an international date format that sorts both numerically and alphabetically.  The shorter name makes it easier to pick out in small message boxes.

**Comment by yaki29** [comment]
@rjt @dessant Thanks for reviewing, I will correct these mistakes.

File: examples/camera/main.py [hunk 4]
```diff
@@ -29,12 +30,29 @@
...
+    Button:
+        text: 'Capture'
+        size_hint_y: None
+        height: '48dp'
+        on_press: root.Capture()
 '''
-
+)
+class CameraClick(BoxLayout):
+    def Capture(self):
```
Comment (dessant) [inline examples/camera/main.py]: Function and method names should usually start with a lower case letter.

**Comment by dessant** [comment]
Please make sure that travis and appveyor builds succeed, you can click on the icon near the commits to get more details.

**Comment by yaki29** [comment]
@dessant Thank you .... Actually I'm new at this .... That was my first PR ... that's why it took that much time. ☺☺

**Pull request merged**